crates/types/src/net/replicated_loglet.rs

```diff
@@ -74,7 +77,7 @@ pub enum SequencerStatus {
     Error { retryable: bool, message: String },
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, bilrost::Message)]
```

Comment:
It's a very good practice to explicitly tag each field in the message and not rely on the field order. This way we can avoid unpredictable issues that can happen if fields were re-ordered (or a new field was added out of order)

Reply:
This applies to all `bilrost::Message` types